docs/docs/installation.md

```diff
@@ -101,7 +101,7 @@ If you use [Create React App](https://github.com/facebookincubator/create-react-
 
 #### Webpack
 
-Include both `DefinePlugin` and `UglifyJsPlugin` into your production Webpack configuration as described in [this guide](https://webpack.js.org/guides/production-build/).
+Include both [`DefinePlugin`](https://webpack.github.io/docs/list-of-plugins.html#defineplugin) and [`UglifyJsPlugin`](https://webpack.github.io/docs/list-of-plugins.html#uglifyjsplugin) into your production Webpack configuration.
```

Comment:
My only worry with that is users clicking to the other linked doc pages from that tutorial and not really realizing they're potentially getting bad information.

We could mention that this article is on the Webpack 2 doc site after linking to it? Something like:

> This guide works with Webpack 1.x and 2, but is hosted on the new Webpack 2 site. If you're not using the Webpack 2 beta, refer to the [Webpack 1.x website](https://webpack.github.io/docs) for all other documentation.

Otherwise, I can just revert it back 👍